feature/networkinstance/feature.textproto

```diff
@@ -33,6 +33,26 @@ telemetry_path {
   path: "/network-instances/network-instance/state/type"
 }
 
+# Static routes
```

Comment:
I am not entirely convinced we should add the static routes to the base network instance feature profile. Shouldn't we create a separate feature profile for static routes, which would be on the same level as other protocols, i.e. bgp, isis? WDYT @dplore ?

Reply:
Done!